connection.go

```diff
@@ -40,6 +41,9 @@ type mysqlConn struct {
 	finished chan<- struct{}
 	canceled atomicError // set non-nil if conn is canceled
 	closed   atomicBool  // set when conn is closed, before closech is closed
+
+	// for connection check support (Go 1.10+)
+	reset bool // set when the Go SQL package calls ResetSession
```

Comment:
move this member above, after `parseTime`.

Reply:
Done!